bin/cli.js

```diff
@@ -40,10 +40,10 @@ yargs.getInstance()
     'Tasks include:',
     yargs.listTasks().join(', ')
   ].join('\n')))
-  .example('$0', 'Interactive menu')
-  .example('$0 -v', 'Display the version of angularity')
-  .example('$0 -h <task name>', 'Get help on a particular task')
-  .example('$0 <task name>', 'Run the given task')
+  .example('angularity', 'Interactive menu')
```

Comment:
The $0 feature of yargs is meant to simplify examples by resolving to the name of the executable but doesn't work properly in Windows.

Expected `angularity` actual `c:/full/path/to/angularity/global/install`.

Instead I have just written `angularity` explicitly.